doc/quickstart/qgis_quickstart.rst

```diff
@@ -150,24 +127,17 @@ but instead will read as 'My QGIS project'.
 Open an existing project
 ================================================================================
 
-Learning goal:
-
-* Understand what a QGIS project is
-* Open an existing project
-* Interact with layers
-
 A QGIS project saves layers, styles, map extents and settings for use at a later time.
 This information is stored as a QGIS project file and has an extension of '.qgs' or '.qgz',
 with '.qgz' being the default file format in QGIS going into the future.
 
 Let's open an existing QGIS project and have a look at how it works.
 
-#. Open QGIS. Note that when QGIS is opened a user will be presented with a list of recent projects to chose from. This is a quick way of entering into recent projects.
-#. Go to the 'Project' menu on the menu bar.
-#. Select 'Open'.
+#. Open QGIS. Note that when QGIS is opened you will be presented with a list of recent projects to chose from. This is a quick way of entering into recent projects.
```

Comment:
This is a quick way of entering into recent projects. -> This is a convenient way to open a project you have used recently.

Reply:
done